### ISSUE TITLE
Error when trying to build

### ISSUE THREAD

**Issue by BachoSeven** [GuardKenzie/miniplayer#24]
Hi, I'm the maintainer of the AUR package; I'm having issues building the package, either with `python setup.py build` or with `python -m build --wheel`:

```
error: Multiple top-level packages discovered in a flat-layout: ['img', 'miniplayer'].

To avoid accidental inclusion of unwanted files or directories,
setuptools will not proceed with this build.

If you are trying to create a single distribution with multiple packages
on purpose, you should not rely on automatic discovery.
Instead, consider the following options:

1. set up custom discovery (`find` directive with `include` or `exclude`)
2. use a `src-layout`
3. explicitly set `py_modules` or `packages` with a list of names

To find more information, look for "package discovery" on setuptools docs.

```

**Comment by GuardKenzie** [comment]
Thanks for letting me know. There appear to have been some changes to package discovery in setuptools which I wasn't aware of. This has been fixed with f7b99d5

**Comment by BachoSeven** [comment]
Thanks!
